src/lib/nlcManager.js

```diff
@@ -29,6 +33,7 @@ function NLCManager(options) {
 	this.opts.classifierName = options.classifierName || 'default-classifier';
 	this.opts.maxClassifiers = options.maxClassifiers || 3;
 	this.opts.classifierLanguage = options.language || 'en';
+	this.opts.saveTrainingData = options.saveTrainingData || true;
```

Comment:
If you pass in options where options.saveTrainingData is false, this will still set saveTrainingData to true.  Need to implement a different way.